composer.json

```diff
@@ -19,6 +19,7 @@
         "ext-json": "*",
         "ext-mbstring": "*",
         "ext-openssl": "*",
+        "doctrine/dbal": "^3.0",
```

Comment:
Although under SemVer it's allowed to bump minimum versions of an external library, we usually don't do this at Laravel because this will prevent everyone on Doctrine v2 from upgrading. Also, Doctrine ORM doesn't supports DBAL v3 yet which makes apps and libraries using ORM unable to upgrade to new Laravel versions.

Reply:
Besides that, doctrine/dbal shouldn't be a requirement as it's only used for a very specific feature in Laravel.

Reply:
Sorry it was added by mistake

Reply:
@driesvints check now

Reply:
@driesvints  But it making `tests / PHP 8.0 - prefer-stable (pull_request) ` test fails cause it needs it.